src/web/components/Core/KeySecretReveal.tsx

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from 'react';
+
+import Popover from './Popover';
+import { StatusPopup } from './StatusPopup';
+
+import './KeySecretReveal.scss';
+
+const MAX_SHOWN_VALUE_LENGTH = 20;
+
+function ViewKeyButton({
+  value,
+  title,
+  confirmCopiedSecret,
+}: {
+  value: string;
+  title: string;
```

Comment:
maybe a different name like `valueName` would better convey the meaning of this prop better than `title`?
e.g. for
```
<h2>Secret</h2>
<KeySecretReveal
  title='Secret'
  value={secrets.secret}
  setCopiedSecrets={setCopiedSecrets}
/>
```
It would seem like `<h2>Secret</h2>` is the title.

Reply:
Yeah that makes sense. That was a leftover from when I had the title `<h2>Secret</h2>` in this component but now that it's not this make far more sense.